tests/Resources/Page/ODH/JupyterHub/AccessGroups.resource

```diff
@@ -0,0 +1,161 @@
+*** Settings ***
+Resource      ../../../Page/Components/Components.resource
+Resource      ../../../Page/OCPDashboard/UserManagement/Groups.robot
+Resource      ../../../Common.robot
+Resource      JupyterHubSpawner.robot
+Resource      HighAvailability.robot
+Library       JupyterLibrary
+Library       OpenShiftCLI
+
+
+*** Variables ***
+${CUSTOM_ADMINS_GROUP}=   custom-admins-group
+${CUSTOM_USERS_GROUP}=    custom-users-group
+${STANDARD_ADMINS_GROUP}=  rhods-admins
+${STANDARD_USERS_GROUP}=   rhods-users
+
+
+*** Keywords ***
+Create Custom Groups
```

Comment:
Admin groups are used in JuputerHub but also in RHODS Dasboard. I think all these keywords are RHODS specific and could go in tests/Resources/ODS.robot